templates/deposit-compliance/index.jsx

```diff
@@ -11,11 +11,27 @@ import {
   CrossRepositoryCheckCard,
 } from './cards'
 
-import { Icon } from 'design'
+import { Icon, Message, Card, Link } from 'design'
 import Title from 'components/title'
 import Markdown from 'components/markdown'
 import { intro as texts } from 'texts/depositing'
 
+const SUPPORT_EMAIL_URL = 'mailto:t%68%65t%65am%40core%2e%61c%2eu%6b'
+const SUPPORT_EMAIL = decodeURIComponent(
+  SUPPORT_EMAIL_URL.slice('mailto:'.length)
+)
+
+const NotEnoughDataMessage = () => (
+  <Card>
+    <Message variant="danger">
+      Your repository is not configured to expose information on dates of
+      deposit in a machine-readable format. For more information check our
+      guidelines and contact us at{' '}
```

Comment:
'Guidelines' needs to be linked. @mcancellieri do you have any idea where we should link it?

Reply:
I would link to thise one. A bit generic but it contains everything https://core.ac.uk/ref-audit/